### PR TITLE
Ticket number 10012 - Cypress_assertions

### PR DESCRIPTION
**Pull request by TetianaZholudova**
Hello Stanislav, 
please take a look and review my PR

### CODE REVIEW

File: cypress/e2e/emailValidation.cy.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    beforeEach(() => {
+    
+      cy.visit('/')
+      cy.get('.hero-descriptor_btn').click()
+  
+    })
+
+it('should show "Email required" when the email field is empty', ()=> {
+    registrationPage.getSignupEmail().click();
+    registrationPageStepsEmailField.leaveEmailFieldEmpty()
```
Comment (sttaran) [inline cypress/e2e/emailValidation.cy.js]: it is overhead

Comment (sttaran) [reply cypress/e2e/emailValidation.cy.js]: for this scenario two cypress methods would be enough 

https://docs.cypress.io/api/commands/blur
https://docs.cypress.io/api/commands/focus

File: cypress/e2e/emailValidation.cy.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+})
+  
+it('should show "Email is incorrect" when the username contains wrong data', ()=> {
+    registrationPage.getSignupEmail().type('tanya.zhol.gmail.com');
+    registrationPageStepsEmailField.enterIncorrectEmail()
```
Comment (sttaran) [inline cypress/e2e/emailValidation.cy.js]: enterIncorrectEmail does not represent what this method actually does

it does not enter anything

it checks validation

**Review by sttaran — CHANGES_REQUESTED** [review]

**Comment by TetianaZholudova** [comment]
Hello Stanislav, 
I have refactored the code according to your comments. Please review and approve or advise on further improvements

**Review by sttaran — APPROVED** [review]

**Pull request merged**